Updated CodePlex link & text

doddag opened the pull request:
# Replace/remove CodePlex links

Replaced the CodePlex text with GitHub
Repalced the CodePlex link with GitHub\Microsoft

as this link https://github.com/Microsoft seems appropriate

## Summary

Updated the CodePlex links/text

## Details

Replaced the CodePlex text with GitHub
Repalced the CodePlex link with GitHub\Microsoft

as this link https://github.com/Microsoft seems appropriate

## Suggested Reviewers

@mairaw

doddag committed:
Updated CodePlex link & text

Replaced the CodePlex text with GitHub
Repalced the CodePlex link with GitHub\Microsoft

as this link https://github.com/Microsoft seems appropriate

dnfclas commented:
This seems like a small (but important) contribution, so no Contribution License Agreement is required at this point. We will now review your pull request.
_Thanks,
.NET Foundation Pull Request Bot_

doddag committed:
Updated the link to Workflow foundation samples

Updated the text from CodePlex to Microsoft Docs
Updated the link from http://aka.ms/WF45Samples to Updated the link to https://docs.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/samples/

Added Sample projects to docs\samples\framework\workflow foundation which i downloaded from https://wf.codeplex.com/releases

1. Microsoft WF Azure Activity Pack CTP 1 Sample
2.  WF Migration Kit CTP 2 Samples

mairaw commented:
Related to issue #2974

mairaw commented:
@doddag I went through the PR and I'm slightly confused by the changes on the second commit. There are a lot of code files being added that are not related to the CodePlex change. Perhaps the easier thing to do is to remove the last commit and continue with the changes to remove CodePlex from our docs. Let me know if you need any help or directions on how to accomplish that. I can also do that on your behalf if you gave maintainers permissions.

My expectation for this PR would be to only touch the files returned by the search I've included in the issue: https://github.com/dotnet/docs/search?utf8=%E2%9C%93&q=codeplex&type=

Thanks so much for working on this issue!

dnfclas commented:
@__doddag__,
Thanks for your contribution.
To ensure that the project team has proper rights to use your work, please complete the Contribution License Agreement at https://cla2.dotnetfoundation.org.

It will cover your contributions to all .NET Foundation-managed open source projects.
_Thanks,
.NET Foundation Pull Request Bot_

mairaw commented:
Closed and reopened to trigger a new CLA validation.

dnfclas commented:
__@doddag__, thanks for signing the contribution license agreement. We will now validate the agreement and then the pull request.
<br />_Thanks, .NET Foundation Pull Request Bot_

doddag committed:
Revert "Updated the link to Workflow foundation samples"

This reverts commit 9c9a7455d151fb4b158f7576952d3cdae799610d.

doddag commented on docs/framework/windows-workflow-foundation/samples/index.md:
```diff
@@ -19,7 +19,7 @@ These samples provide instruction on various aspects of [!INCLUDE[wf](../../../.
   
 > [!NOTE]
->  These samples were created with [!INCLUDE[vs2010](../../../../includes/vs2010-md.md)] and [!INCLUDE[netfx40_short](../../../../includes/netfx40-short-md.md)], but are compatible with [!INCLUDE[vs_current_long](../../../../includes/vs-current-long-md.md)] and [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)]. Additional samples for [!INCLUDE[wf2](../../../../includ ...
+>  These samples were created with [!INCLUDE[vs2010](../../../../includes/vs2010-md.md)] and [!INCLUDE[netfx40_short](../../../../includes/netfx40-short-md.md)], but are compatible with [!INCLUDE[vs_current_long](../../../../includes/vs-current-long-md.md)] and [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)]. Additional samples for [!INCLUDE[wf2](../../../../includ ...
```
I have updated the aka.ms to https://code.msdn.microsoft.com/site/search?f%5B0%5D.Type=Technology&f%5B0%5D.Value=Windows%20Workflow%204.5

Will do a commit shortly

thank you

doddag commented:
Hi I have reverted my 2 nd commit
and updated the Work flow foundation code samples link  to https://code.msdn.microsoft.com/site/search?f%5B0%5D.Type=Technology&f%5B0%5D.Value=Windows%20Workflow%204.5

doddag committed:
Updated the Text & link related to CodePlex

https://code.msdn.microsoft.com/site/search?f%5B0%5D.Type=Technology&f%5B0%5D.Value=Windows%20Workflow%204.5

mairaw commented on docs/framework/windows-workflow-foundation/samples/index.md:
```diff
@@ -19,7 +19,7 @@ These samples provide instruction on various aspects of [!INCLUDE[wf](../../../.
   
 > [!NOTE]
->  These samples were created with [!INCLUDE[vs2010](../../../../includes/vs2010-md.md)] and [!INCLUDE[netfx40_short](../../../../includes/netfx40-short-md.md)], but are compatible with [!INCLUDE[vs_current_long](../../../../includes/vs-current-long-md.md)] and [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)]. Additional samples for [!INCLUDE[wf2](../../../../includ ...
+>  These samples were created with [!INCLUDE[vs2010](../../../../includes/vs2010-md.md)] and [!INCLUDE[netfx40_short](../../../../includes/netfx40-short-md.md)], but are compatible with [!INCLUDE[vs_current_long](../../../../includes/vs-current-long-md.md)] and [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)]. Additional samples for [!INCLUDE[wf2](../../../../includ ...
```
Microsoft Code -> Code MSDN

no need to update the link. I'll get someone to update the aka.ms link

mairaw replied:
We have updated the aka.ms link so we just need to update the link text here

mairaw commented on docs/csharp/getting-started/additional-resources.md:
```diff
@@ -45,8 +45,8 @@ The following sites can help you find answers to common problems.
   
- [CodePlex](http://go.microsoft.com/fwlink/?LinkId=137330)  
- Hosts open source software projects. You can use CodePlex to create new projects to share with the world or join others who have already started their own projects.  
+ [Microsoft](https://github.com/Microsoft)  
```
I agree with you @svick. Wondering if we should just pull this link out from here or even kill this page on a separate PR. @BillWagner?

BillWagner replied:
I think we should kill this page. It really doesn't serve its original purpose anymore.

mairaw committed:
feedback

mairaw committed:
redirection for additional resources page

mairaw committed:
Delete additional-resources.md
